docs/user/customization.md

```diff
@@ -142,6 +142,53 @@ pullSecret: '{"auths": ...}'
 sshKey: ssh-ed25519 AAAA...
 ```
 
+### Image content sources
+
+An example install config with custom image content sources:
+
+```yaml
+apiVersion: v1
+baseDomain: example.com
+metadata:
+  name: test-cluster
+imageContentSources:
+- mirrors:
+  - registry.example.com/ocp4/openshift4
+  source: quay.io/openshift-release-dev/ocp-release-nightly
+- mirrors:
+  - registry.example.com/ocp4/openshift4
+  source: quay.io/openshift-release-dev/ocp-v4.0-art-dev
+platform: ...
+pullSecret: '{"auths": ...}'
+sshKey: ssh-ed25519 AAAA...
+```
+
+That configuration is compatible with mirrored releases created with `mirror` commands like:
+
+```console
+$ oc adm release mirror \
+>   --from=quay.io/openshift-release-dev/ocp-release-nightly:4.2.0-0.nightly-2019-09-11-114314 \
```

Comment:
can we fuzz this `4.2.0-0.nightly-2019-09-11-114314` from so that people don't try to use this when copy-paste

Reply:
I tried to fuzz it with "like" when introducing it ;).  Should I keep `-nightly`?  Currently those are the only public release images that support mirrored pulls.  If I keep them, do you just want `--from=quay.io/openshift-release-dev/ocp-release-nightly:"${YOUR_CHOSEN_TAG}"`?

Reply:
Hrm, and then should I fuzz the output I quote below and the YAML I mention above for the install-config?

Reply:
Maybe I can have:

```console
$ YOUR_CHOSEN_TAG=4.2.0-0.nightly-2019-09-11-114314  # chose the one you want to mirror
$ oc adm release mirror \
>   --from=quay.io/openshift-release-dev/ocp-release-nightly:"${YOUR_CHOSEN_TAG}"
...
```

Reply:
or without the variable:

```console
$ oc adm release mirror \
>   --from=quay.io/openshift-release-dev/ocp-release-nightly:4.2.0-0.nightly-2019-09-11-114314 
 `# choose the release you want to mirror` \
>   --to=registry.example.com/ocp4/openshift4 `# choose the repository you want to push to` \
>   --to-release-image=registry.example.com/ocp4/openshift4:4.2.0-0.nightly-2019-09-11-114314 # choose the the pullspec you want to push to
```

Using [the cute process-substitution hack][1] for interspersing comments in multi-line commands :p.

[1]: https://stackoverflow.com/a/12797512

Reply:
> Using the cute process-substitution hack for interspersing comments in multi-line commands :p.

I think `oc adm release mirror --help` should cover that.. 

> YOUR_CHOSEN_TAG

I think if you use `XXXXXX` that's fine too, just fuzzing it enough so that people look at it when copy-pasting...

Reply:
> I think if you use XXXXXX that's fine too...

Done with dbf80fc91 -> 7f10648b5.